(core) - Expose client.subscription shortcut method (#838)
FormidableLabs/urql: .changeset/hot-dryers-drive.md

---
'@urql/core': minor
---

> Expose `client.subscription` to be correctly recognized by exchange

Reviewer: ```suggestion
Expose a `client.subscription`shortcut method, similar to `client.query` and `client.mutation`
```
